assignment/scripts/loops-practice.js

```diff
@@ -14,17 +14,35 @@ for (let i=0; i<4; i++) {
 //   - Which part of the example loop do you need to change to do this?
 console.log('count from 0 to 5');
 
+for (let i=0; i<5; i++) {  
+  console.log(i);
+}
+
 // 1.b. TODO: Write a for loop to console.log the numbers from 3 to 5
 //   - Which part of the example loop do you need to change to do this?
 console.log('count from 3 to 5');
 
+for (let i=3; i<6; i++) {  
```

Comment:
Would love to see spaces around these operators (other than `++`).

```suggestion
for (let i = 3; i < 6; i++) {  
```